foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-APIManagementPolicySetDefinition.json

```diff
@@ -84,6 +84,328 @@
+                                                    "where": {
+                                                        "field": "Microsoft.ApiManagement/service/additionalLocations[*].publicIpAddressId",
+                                                        "exists": "true"
+                                                    }
+                                                },
+                                                "greater": 0
+                                            }
+                                        ]
+                                    }
+                                ]
+                            },
+                            "then": {
+                                "effect": "[[parameters('effect')]"
+                            }
+                        }
+                    },
+                    "name": "Deny-APIM-PublicIpAddress"
+                },
+                {
+                    "properties": {
+                        "displayName": "API Management should restrict virtual network type",
+                        "mode": "Indexed",
+                        "description": "Azure API Management should use internal virtual network type for enhanced security, isolation and prohibiting public network traffic. The API service then only allows connections via the connected network.",
+                        "metadata": {
+                            "version": "1.0.1",
+                            "category": "API Management"
+                        },
+                        "parameters": {
+                            "effect": {
+                                "type": "String",
+                                "metadata": {
+                                    "displayName": "Effect",
+                                    "description": "Enable or disable the execution of the policy"
+                                },
+                                "allowedValues": [
+                                    "Audit",
+                                    "Deny",
+                                    "Disabled"
+                                ],
+                                "defaultValue": "Deny"
+                            },
+                            "evaluatedSkuNames": {
+                                "type": "Array",
+                                "metadata": {
+                                    "displayName": "API Management SKU Names",
+                                    "description": "List of API Management SKUs against which this policy will be evaluated."
+                                },
+                                "allowedValues": [
+                                    "Developer",
+                                    "Basic",
+                                    "Standard",
+                                    "Premium",
+                                    "Consumption"
+                                ],
+                                "defaultValue": [
```

Comment:
Same as the previous comment on SKUs

Reply:
This is correct, as it only applies to these SKUs. Others cannot be connected to the vnet.